docs/index.md

```diff
@@ -6,11 +6,11 @@ hide:
 
 # Browsertrix Docs
 
-Browsertrix is an open source web archiving system by [Webrecorder](https://webrecorder.net/). Sign up for Browsertrix on [browsertrix.com](https://browsertrix.com/) to start archiving with zero setup, or follow our self-hosting guide.
+Browsertrix is an open source web archiving system created by [Webrecorder](https://webrecorder.net/). Sign up for Browsertrix on [browsertrix.com](https://browsertrix.com/) to start archiving with zero setup, or follow our self-hosting guide to deploy Browsertrix on your own infrastructure.
 
 ## Documentation
 
-Browsertrix Docs are organized into the following sections. If you have feedback on the docs, please feel free to [reach out to us](mailto:docs-feedback@webrecorder.net).
+Browsertrix' Docs are organized into the following sections. If you have feedback on the docs, please feel free to [reach out to us](mailto:docs-feedback@webrecorder.net).
```

Comment:
Assuming this is a typo @Shrinks99 ?

Reply:
Yeah, should be `Browsertrix's`.  I personally drop the s if there's an x too sometimes but I'm pretty sure that's wrong.

Added apostrophe to signify that the docs belong to "Browsertrix".